asab/log.py

```diff
@@ -155,6 +155,8 @@ async def schedule(app, interval):
 			level = logging.getLevelName(levelname.upper())
 			logging.getLogger(loggername).setLevel(level)
 
+		# adding placeholder for LogCounter - see native metrics
+		self.RootLogger.manager.LogCounter = None
```

Comment:
`manager`? ... is that a naming crisis?

Reply:
That's not my manager! That's python's manager! I just used it. :D

Reply:
Okej ;-)